### PR TITLE
Addressing git issue for darkmode tweaks

### PR DESCRIPTION
**Pull request by klr981**
Closes https://github.com/dart-lang/dartdoc/issues/3116

### After tweaks

Screenshot 2022-08-24 12 40 45 PM
Screenshot 2022-08-24 12 40 37 PM

### CODE REVIEW

**Comment by srawlins** [comment]
CC @mit-mit @parlough regarding colors

File: lib/templates/html/_head.html [hunk 1]
```diff
@@ -1,9 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
-  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
-  <meta charset="utf-8">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />  <meta charset="utf-8">
```
Comment (parlough) [inline lib/templates/html/_head.html:4]: Can you move this `link` after the Google fonts preconnect and next to the other included fonts? Around line 26-27. That makes it a bit more consistent and easy to find.

**Review by parlough — APPROVED** [review]
The simpler light/dark mode switch is great and the background changes look to be what is desired.

It seems a bit strange to use the same color for the code block, but for now there is the outline and it does improve contrast which is nice.

File: lib/templates/html/_head.html (hunk 1 above)
Comment (klr981) [reply lib/templates/html/_head.html]: Addressed in new commit

File: lib/resources/styles.css [hunk 2]
```diff
@@ -73,87 +74,16 @@
...
-/* Rounded sliders */
-.slider.round {
-  border-radius: 34px;
-}
-
-.slider.round:before {
-  border-radius: 50%;
+input[type=checkbox] {
+  display: none;
 }
```
[the remark this answers is not in the document]
Comment (klr981) [reply lib/resources/styles.css]: Addressed in new commit

**Review by keertip — APPROVED** [review]

**Review by mit-mit — APPROVED** [review]
This looks much better, thanks!

**Pull request merged**